src/test/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanelTest.kt

```diff
@@ -144,7 +144,7 @@ class SnykToolWindowPanelTest : LightPlatform4TestCase() {
     fun `should automatically enable all products on first run after Auth`() {
         every { snykApiServiceMock.getSastSettings() } returns CliConfigSettings(
             true,
-            LocalCodeEngine(false),
+            LocalCodeEngine(true, "http://foo.bar/api", false),
```

Comment:
This is now missing a test for the "standard" Snyk Code enablement - would you please have both tested?

Reply:
Done